tools/checkpoints-tool/src/main.rs

```diff
@@ -59,32 +59,60 @@ struct Cli {
     /// the latest finality update is used to get the slot.
     #[arg(long, env = "SLOT_CHECKPOINT")]
     slot_checkpoint: Option<u64>,
+
+    /// Skip code upload if the code already exists (prevents duplicate uploads)
+    #[arg(long, default_value_t = false, env = "SKIP_CODE_UPLOAD")]
+    skip_code_upload: bool,
+
+    /// Specify salt for the send_recv call (hex string)
+    #[arg(long, env = "SALT")]
+    salt: Option<String>,
+
+    /// Specify the code ID to use when skipping code upload (hex string)
+    #[arg(long, env = "CODE_ID")]
+    code_id: Option<String>,
```

Comment:
Nope, we don't need that. Instead determine the code id from the wasm binary like there - https://github.com/gear-tech/gear-bridges/blob/main/tools/deploy-to-gear/src/main.rs#L246